starcheck/src/lib/Ska/Parse_CM_File.pm

```diff
@@ -747,6 +747,11 @@ sub MM {
         $manvr_hash{q2} = $quat[1];
         $manvr_hash{q3} = $quat[2];
         $manvr_hash{q4} = $quat[3];
+        my @init_quat = ($1,$2,$3,$4) if ($initial_attitude =~ /Quaternion:\s+(\S+)\s+(\S+)\s+(\S+)\s+(\S+)/);
```

Comment:
I don't recall if I had a reason to directly calculate the maneuver angle; else we can just use the angle from the maneuver summary too.